test/verify/check-loopback

```diff
@@ -17,36 +17,50 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+import os
+
 import parent  # noqa: F401
-from testlib import MachineCase, skipDistroPackage, skipOstree, test_main
+from testlib import MachineCase, nondestructive, skipDistroPackage, skipOstree, test_main
 
 
 @skipOstree("OSTree always uses loopback")
 @skipDistroPackage()
+@nondestructive
 class TestLoopback(MachineCase):
 
     def testBasic(self):
         b = self.browser
         m = self.machine
 
         # Start Cockpit with option to access SSH
-        m.spawn(f"{self.libexecdir}/cockpit-ws --local-ssh --no-tls", "cockpit-ws")
+        m.stop_cockpit()
+        pid = m.spawn(f"{self.libexecdir}/cockpit-ws --local-ssh --no-tls", "cockpit-ws")
+        self.addCleanup(m.execute, "kill %i" % pid)
         m.wait_for_cockpit_running()
 
         b.login_and_go("/system", user="admin")
         b.logout()
         b.wait_visible("#login")
 
-        m.execute("while B=$(command -v cockpit-bridge); do rm $B; done")
-        b.set_val('#login-user-input', "admin")
-        b.set_val('#login-password-input', "foobar")
-        b.click('#login-button')
-        b.wait_visible('#login-fatal')
-        self.assertIn("no-cockpit", b.text('#login-fatal'))
+        try:
+            m.execute("mv /usr/bin/cockpit-bridge /usr/bin/cockpit-bridge.disabled")
```

Comment:
would not it be cleanerto just use restore_flie for these files in the start of the test? To avoid the exception handling.

Reply:
You are right -- initially I thought the second half of the test would need the bridge back, but it doesn't -- it doesn't get past SSH authentication. Simplified.